ajax/libs/jshashtable.js/3.0/hashset.js

```diff
@@ -0,0 +1,119 @@
+/**
```

Comment:
This file's md5 hash is `3f5549207b6eb75540787ecf1de294a1`
But the origin hash is `cfe58d496d66dfe97c5af27c2aad331a`
Would you please re-add this file and generate the minified files again?
Thank you.